docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md

```diff
@@ -0,0 +1,332 @@
+
+- A version control system (VCS) such as GitHub or GitLab.
+- An existing pipeline or a plan to set one up using tools like [CircleCI](https://circleci.com/) or [Jenkins](https://www.jenkins.io/), cloud platforms such as [Azure DevOps Pipelines](https://azure.microsoft.com/de-de/products/devops), or built-in VCS platforms like [GitHub Actions](https://github.com/features/actions) or [GitLab's DevSecOps Lifecycle](https://about.gitlab.com/stages-devops-lifecycle/).
+- Make yourself familiar with the [Web Modeler API](/apis-tools/web-modeler-api/index.md) through the [OpenAPI documentation](https://modeler.cloud.camunda.io/swagger-ui/index.html).
+- Understand how [clusters](http://localhost:3000/docs/next/components/concepts/clusters/) work in Camunda Platform 8.
+- Ensure you’ve [created a Camunda Platform 8 account](/guides/create-account.md), or installed [Camunda Platform 8 Self-Managed](/self-managed/about-self-managed.md).
+
+## Setup
+
+While a pipeline for process application integration and deployment resembles general software CI/CD pipelines, key distinctions exist. Consider the following:
+
+- Web Modeler uses [milestones](/components/modeler/web-modeler/milestones.md) to indicate specific process states, such as readiness for developer handover, review, or deployment.
+- A process application comprises main processes and diverse resources, such as sub processes, forms, DMN decision models, Connectors, job workers, and orchestrated services. Some applications bundle these resources, while others focus on a single process for deployment.
+- Process reviews differ from code reviews, occurring on visual diagrams rather than XML.
+
+![Sample CI/CD setup with Web Modeler](img/modeler-ci-cd.png)
+
+### Obtain API clients and tokens
+
+Before getting started, obtain API clients and tokens for integrating Web Modeler and accessing the process engine via API:
+
+- [Obtain an API token for Web Modeler](/apis-tools/web-modeler-api/index.md#authentication)
+- [Obtain an API client for Zeebe](/guides/setup-client-connection-credentials.md)
+
+### Disable manual deployments from Web Modeler
+
+To enforce pipeline-driven deployments to your environments, consider disabling manual deployments.
+
+<Tabs groupId="disableDeployments" defaultValue="sm" values={[{label: 'Self-Managed', value: 'sm', }, {label: 'SaaS', value: 'saas', },]} >
+<TabItem value="sm">
+
+You can completely disable manual deployments by configuring environment variables `ZEEBE_BPMN_DEPLOYMENT_ENABLED` and `ZEEBE_DMN_DEPLOYMENT_ENABLED` as documented [here](/self-managed/modeler/web-modeler/configuration/configuration.md/#general).
+
+</TabItem>
+<TabItem value="saas">
+
+Currently, manual deployments are unrestricted on SaaS. Efforts are underway to introduce permissions for restricting deployments.
```

Comment:
We could mention here that only the `Developer` role allows deployments from Web Modeler. Assigning any other role effectively removes deployment privileges, see 

https://github.com/camunda/camunda-platform-docs/blob/01845885995f2a91908aeeb88dccc7dd0a9483ba/docs/components/console/manage-organization/manage-users.md?plain=1#L25-L29

Reply:
You're right! Totally missed that. Thanks!